doc/rest-api.md

```diff
@@ -764,6 +773,83 @@ Input (none at present):
 
 HTTP code for this should be 202 (Accepted).
 
+## `/1.0/containers/<name>/backups`
```

Comment:
You forgot to add the backup endpoints to the full list of endpoints at the top of this document.